logging/tests/unit/handlers/transports/test_sync.py

```diff
@@ -14,6 +14,8 @@
 
 import logging
 import unittest
+from google.cloud.logging.resource import Resource
```

Comment:
It's counter-intuitive, but the testing style in this project is to import package-local modules within the tests functions and not globally.